tools/test.txt

```diff
@@ -0,0 +1 @@
+bar
```

Comment:
Could you please remove this file? It's unnecessary :)

Reply:
Completely forgot to remove it during testing, my bad! Will do so